client/src/Movies/Movie.js

```diff
@@ -5,13 +5,13 @@ export default class Movie extends Component {
   constructor(props) {
     super(props);
     this.state = {
-      movie: null
+      movie: props.match.arguments
     };
   }
 
   componentDidMount() {
     // change this line to grab the id passed on the URL
-    const id = 1;
+    const id = this.props.match.arguments;
```

Comment:
In order to fetch a dynamic parameter from the url, you have to drill down the props like `this.props.match.params.id`